packages/odf/components/mcg/replication-policy-form.tsx

```diff
@@ -129,80 +132,114 @@ export const ReplicationPolicyForm: React.FC<ReplicationFormProps> = ({
     updateParentState(rules);
   };
 
-  const hasSyncDeletion = rules.some((rule) => rule.syncDeletion);
+  const disableSyncDeletions = (val: boolean) => {
```

Comment:
```suggestion
  const disableSyncDeletions = (val: boolean) => {
```
can we use a better name than `val`?

Reply:
changed to isEventLogEnabled